dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/submit/DescribeStep.java

```diff
@@ -416,893 +407,900 @@ else if (inputType.equals("qualdrop_value"))
-        {
-                // The date field consists of three primitive fields: a text field
-                // for the year, followed by a select box of the months, follewed
-                // by a text box for the day.
-                Composite fullDate = form.addItem().addComposite(fieldName, "submit-date");
-                Text year = fullDate.addText(fieldName+"_year");
-                Select month = fullDate.addSelect(fieldName+"_month");
-                Text day = fullDate.addText(fieldName+"_day");
-
-                // Set up the full field
-                fullDate.setLabel(dcInput.getLabel());
-                fullDate.setHelp(cleanHints(dcInput.getHints()));
-                if (dcInput.isRequired())
-                {
-                    fullDate.setRequired();
-                }
-                if (isFieldInError(fieldName))
+        if (dcInput.isRepeatable() && !readonly)
+        {
+            fullName.enableAddOperation();
+        }
+        if ((dcInput.isRepeatable() || dcValues.size() > 1) && !readonly)
+        {
+            fullName.enableDeleteOperation();
+        }
+        String fieldKey = metadataAuthorityService.makeFieldKey(dcInput.getSchema(), dcInput.getElement(), dcInput.getQualifier());
+        boolean isAuthorityControlled = metadataAuthorityService.isAuthorityControlled(fieldKey);
+        if (isAuthorityControlled)
+        {
+            fullName.setAuthorityControlled();
+            fullName.setAuthorityRequired(metadataAuthorityService.isAuthorityRequired(fieldKey));
+        }
+        if (choiceAuthorityService.isChoicesConfigured(fieldKey))
+        {
+            fullName.setChoices(fieldKey);
+            fullName.setChoicesPresentation(choiceAuthorityService.getPresentation(fieldKey));
+            fullName.setChoicesClosed(choiceAuthorityService.isClosed(fieldKey));
+        }
+
+        // Setup the first and last name
+        lastName.setLabel(T_last_name_help);
+        firstName.setLabel(T_first_name_help);
+
+        if (readonly)
+        {
+            lastName.setDisabled();
+            firstName.setDisabled();
+            fullName.setDisabled();
+        }
+
+        // Setup the field's values
+        if (dcInput.isRepeatable() || dcValues.size() > 1)
+        {
+            for (MetadataValue dcValue : dcValues)
+            {
+                DCPersonName dpn = new DCPersonName(dcValue.getValue());
+
+                lastName.addInstance().setValue(dpn.getLastName());
+                firstName.addInstance().setValue(dpn.getFirstNames());
+                Instance fi = fullName.addInstance();
+                fi.setValue(dcValue.getValue());
+                if (isAuthorityControlled)
                 {
-                    if (dcInput.getWarning() != null && dcInput.getWarning().length() > 0)
+                    if (dcValue.getAuthority() == null || dcValue.getAuthority().equals(""))
```

Comment:
It's not clear to me here why you changed "getWarning" to "getAuthority" and how this is related to DS-3598.